app/src/release/java/io/github/droidkaigi/confsched2017/debug/ClearCache.java

```diff
@@ -0,0 +1,15 @@
+package io.github.droidkaigi.confsched2017.debug;
+
+import com.tomoima.debot.strategy.DebotStrategy;
+
+import android.app.Activity;
+import android.support.annotation.NonNull;
+
+
+public class ClearCache extends DebotStrategy {
+
+    @Override
```

Comment:
CI is failed because Dagger can not inject `ClearCache`.
Maybe you have to add Constructor with `@Inject` annotation.

```
@Inject
public ClearCache() {
}
```